tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -22,3 +22,11 @@
       name: '{{ pkg }}'
       state: present
       update_cache: yes
+
+- name: set skuba-update to annotate-only mode
+  copy:
+    src: skuba-update
```

Comment:
- This file is missing in the PR
- Can we use https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html instead of a file and just modify the single line? That way, we don't have to check/update the whole file with every new CaaSP version

Reply:
- I added the forgotten file sorry about that.

- I initially wanted to implement it this way, but we do not know when the file is present, because this is skuba that is installing packages on nodes.
We want to avoid skuba-update process completely, so there is no other way than having the proper config already set when the process launches. Otherwise there is a chance that skuba-update is triggered before we actually modifiy its config.


    This file is missing in the PR
    Can we use https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html instead of a file and just modify the single line? That way, we don't have to check/update the whole file with every new CaaSP version